### DISCUSSION TITLE
dynamic Inventory in ansible tower is not working after update to netbox 3.5

### DISCUSSION THREAD

**Discussion by sthegger** [netbox-community/ansible_modules#1005] [Q&A]
We set up a dynamic inventory with netbox and all was fine. Then we updated netbox to the newest release and since then, the dynamic inventory stopped working. Did someone has the same issue and knows how to solve it? I do not find anything in the release notes. Does somebody knows where to look for the problem?

**Comment by sc68cal** [comment]
Did you update the version of `ansible_modules` that your ansible tower install is using? They need to match between the NetBox server version and the version of `ansible_modules`. Specifically, you need to make sure the version of `pynetbox` that is being installed lines up with the NetBox server version.

Especially since #999 was just merged last week, so that we can actually work with NetBox 3.5

**Reply by sthegger** [reply]
We followed your link. We had to build a new execution environment. The one we used was outdated (docker container) We had to build our own docker container with the new actual excecution environment.
